output.proto

```diff
@@ -70,8 +70,13 @@ message SystemHealth {
       string topic = 2;
       string sensor_name = 3;
     }
+    message EdgeNodeInfo {
+      Status status = 1;
+      map<string, SensorInfo> sensors = 2;
+    }
     Status status = 1;
     map<string, SensorInfo> sensors = 4;
+    map<string, EdgeNodeInfo> edges = 5;
```

Comment:
Maybe, it should be named `edge_nodes`. Otherwise, this is the only place we refer to it as `edges` as far as I know.
I am fine with both.

Reply:
I completely agree with you. I tried `edge_nodes` first, but it is converted as `edgeNodes` for json REST API respond, which did not get well with other keys personally. Therefore, I chose `edge` as an alternative. :crying_cat_face:

Reply:
Ah, I see.
Then let it be `edges` :grin: